test/services/dumpers/agent-nodejs/agent-nodejs.test.ts

```diff
@@ -8,14 +8,15 @@ import rimraf from 'rimraf';
 
 import defaultPlan from '../../../../src/context/plan';
 import AgentNodeJs from '../../../../src/services/dumpers/agent-nodejs';
+import Languages from '../../../../src/utils/languages';
```

Comment:
I have the feeling that we could do better on this file. 

In many places we are still asserting stuffs on static files (search for `index.js`) and my recommandation would be that : Anything that is true for js, should be true for ts as-well.

So we might want to put the whole test suits in your loops, to ensure that the tests are also being run on ts (tests will be stronk)